### PR TITLE
CLC-5439 - Apply adjustments discovered from web accessibility review

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.ets.berkeley.edu/jira/browse/CLC-5439

### CODE REVIEW

File: src/assets/templates/canvas_embedded/course_add_user.html [hunk 1]
```diff
@@ -44,6 +44,10 @@ <h1 class="bc-page-course-add-user-header">Find a Person to Add</h1>
           Please refine your search to limit the number of results.
         </div>
 
+        <div data-ng-show="userSearchResultsCount == userSearchResults.length" class="cc-visuallyhidden">
```
Comment (christianv) [inline src/assets/templates/canvas_embedded/course_add_user.html]: @redconfetti `===`

File: src/assets/templates/canvas_embedded/course_add_user.html [hunk 2]
```diff
@@ -142,75 +143,81 @@ <h1 class="bc-page-course-add-user-header">Find a Person to Add</h1>
...
+                  <th scope="col">Name</th>
+                  <th scope="col">Calnet UID</th>
+                  <th scope="col">Email</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr data-ng-repeat="user in userSearchResults">
+                  <td>
+                    <input id="bc-user-search-result-{{$index}}" type="radio" name="selectedUser" data-ng-model="$parent.selectedUser"
+                      data-ng-value="user" data-ng-required="true" aria-labelled-by="bc-user-search-result-{{$index}} bc-user-search-result-ldap-uid-{{$index}}">
```
Comment (christianv) [inline src/assets/templates/canvas_embedded/course_add_user.html]: @redconfetti can we use `ng-attr` for `aria-labelled-by`?

File: src/assets/templates/canvas_embedded/course_add_user.html [hunk 3]
```diff
@@ -142,75 +143,81 @@ <h1 class="bc-page-course-add-user-header">Find a Person to Add</h1>
...
+                  </td>
+                  <td>
+                    <label for="bc-user-search-result-{{$index}}" class="bc-form-input-label-no-align">
```
Comment (christianv) [inline src/assets/templates/canvas_embedded/course_add_user.html]: @redconfetti we should probably use `ng-attr` here as well

File: src/assets/templates/canvas_embedded/course_add_user.html [hunk 4]
```diff
@@ -142,75 +143,81 @@ <h1 class="bc-page-course-add-user-header">Find a Person to Add</h1>
...
+                      <span data-ng-bind="user.firstName + ' ' + user.lastName"></span>
+                    </label>
+                  </td>
+                  <td>
+                    <span id="bc-user-search-result-ldap-uid-{{$index}}">
```
Comment (christianv) [inline src/assets/templates/canvas_embedded/course_add_user.html]: @redconfetti we should probably use `ng-attr` here as well

File: src/assets/templates/canvas_embedded/create_course_site.html [hunk 5]
```diff
@@ -137,7 +137,7 @@ <h3 class="bc-sections-course-title">
                         <span data-ng-if="course.title" data-ng-bind-template=" : {{course.title}}"></span>
                       </h3>
                       <span data-ng-if="course.sections">
-                        (<span data-ng-pluralize count="course.sections.length" when="{'one': '1 section', 'other': '{{course.sections.length}} sections'}"></span>)
+                        &nbsp;(<span data-ng-pluralize count="course.sections.length" when="{'one': '1 section', 'other': '{{course.sections.length}} sections'}"></span>)
```
Comment (christianv) [inline src/assets/templates/canvas_embedded/create_course_site.html]: @redconfetti interesting, why did we need the `&nbsp`?

Comment (redconfetti) [inline src/assets/templates/canvas_embedded/create_course_site.html]: @christianv I could change the formatting of the HTML to make sure that the space is recognized, or I can leave the contents on it's own line for readability. The nbsp character makes that possible.

Comment (christianv) [inline src/assets/templates/canvas_embedded/create_course_site.html]: @redconfetti both solutions are fine with me, thanks for explaining!

**Pull request merged**